bftengine/tests/bcstatetransfer/source_selector_test.cpp

```diff
@@ -33,6 +34,7 @@ constexpr auto kBigDeltaTime = 10 * kSampleCurrentTimeMs;
 constexpr uint16_t kSampleReplicaId = 1;
 
 const auto replicas = std::set<uint16_t>{1, 2, 3};
+concordMetrics::Component metricsComponent("source-selector-test", std::make_shared<concordMetrics::Aggregator>());
```

Comment:
Fixture is ctor/dtor for each test case.
both lines should be there (1 was old, one is new).
Try to insert into ctor or setUp of fixture, that's the right way to do.